src/tiledb/cloud/vcf/utils.py

```diff
@@ -0,0 +1,127 @@
+import os
+import subprocess
+
+import tiledb
+from tiledb.cloud.utilities import process_stream
+
+
+def find_index(vcf_uri: str) -> str:
+    """
+    Find the index file for a VCF file or an empty string if not found.
+
+    :param vcf_uri: URI of the VCF file
+    :return: URI of the index file
+    """
+
+    for ext in ["tbi", "csi"]:
+        index = f"{vcf_uri}.{ext}"
+        if tiledb.VFS().is_file(index):
+            return index
+    return ""
+
+
+def is_bgzipped(vcf_uri: str) -> bool:
+    """
+    Returns True if the VCF file is bgzipped.
+
+    :param vcf_uri: URI of the VCF file
+    :return: True if the VCF file is bgzipped
+    """
+
+    cmd = "file -b -"
+    stdout, stderr = process_stream(vcf_uri, cmd, read_size=1024)
+    if stderr:
+        raise RuntimeError(f"Failed to check file type: {stderr}")
+    return "BGZF" in stdout
+
+
+def get_sample_name(vcf_uri: str) -> str:
+    """
+    Returns the sample name in a VCF file.
+
+    If there are multiple samples, return a comma-separated list of sample names.
+
+    :param vcf_uri: URI of the VCF file
+    :return: sample name
+    """
+
+    cmd = "bcftools query -l"
```

Comment:
e.g. here `('bcftools', 'query', '-l')`

Reply:
Changing all of these commands to lists.